gsa/src/web/pages/tickets/listpage.js

```diff
@@ -80,6 +81,7 @@ const Page = ({
         onFilterChanged={onFilterChanged}
         onInteraction={onInteraction}
         onTicketClose={close}
+        onTicketDelete={deleteticket}
```

Comment:
Please use camelCase here, as well as in line 68

Reply:
fixed